BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBank.swift

```diff
@@ -173,20 +183,20 @@ import GiniCaptureSDK
      - returns: A presentable view controller.
      */
     public class func viewController(withDelegate delegate: GiniCaptureDelegate,
-                                           importedDocument: GiniCaptureDocument? = nil,
-                                           trackingDelegate: GiniCaptureTrackingDelegate? = nil) -> UIViewController {
+                                     importedDocument: GiniCaptureDocument?,
```

Comment:
If it is optional and it has no value assigned to it, it will be nil by default. That is a swiftlint warning if you leave it. 
`- redundant_optional_initialization                 # Initializing an optional variable with nil is redundant.`